patches/tModLoader/Terraria/Player.cs.patch

```diff
@@ -3692,6 +3692,15 @@
  					if (buffType[i] >= 170 && buffType[i] <= 172)
  						DelBuff(i);
  				}
+@@ -38234,7 +_,7 @@
+ 			flag &= (list.Count <= 1);
+ 			flag = (flag && flag2);
+ 			if (flag) {
+-				flag = CheckMana(20, pay: true);
++				flag = CheckMana(new Item(), 20, pay: true);
```

Comment:
We should do something similar to what we did with `GetOtherDamage` in #654, where it doesn't necessarily come from an item. At the least, a string should be used to identify the 'source' of the mana usage.